README.md

```diff
@@ -13,7 +13,8 @@ For Documentation on useing the extension, see [DETAILS.md](DETAILS.md)
 The latest stable version can be found on the
 [Visual Studio Marketplace][Marketplace].
 
-[TODO(przybjw)]:#(change to CI build server)
+[TODO(przybjw)]: # (change to CI build server)
```

Comment:
In general, I feel it might be odd to have your name or LDAP on an open source project. Theoretically everybody can see your name here.